exercises/rna-transcription/RnaTranscriptionTest.cs

```diff
@@ -1,34 +1,54 @@
+// This file was auto-generated based on version 1.0.1 of the canonical data.
+
 using Xunit;
 
-public class ComplementTest
+public class RnaTranscriptionTest
 {
     [Fact]
     public void Rna_complement_of_cytosine_is_guanine()
     {
-        Assert.Equal("G", Complement.OfDna("C"));
+        Assert.Equal("G", RnaTranscription.ToRna("C"));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Rna_complement_of_guanine_is_cytosine()
     {
-        Assert.Equal("C", Complement.OfDna("G"));
+        Assert.Equal("C", RnaTranscription.ToRna("G"));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Rna_complement_of_thymine_is_adenine()
     {
-        Assert.Equal("A", Complement.OfDna("T"));
+        Assert.Equal("A", RnaTranscription.ToRna("T"));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Rna_complement_of_adenine_is_uracil()
     {
-        Assert.Equal("U", Complement.OfDna("A"));
+        Assert.Equal("U", RnaTranscription.ToRna("A"));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void Rna_complement()
     {
-        Assert.Equal("UGCACCAGAAUU", Complement.OfDna("ACGTGGTCTTAA"));
+        Assert.Equal("UGCACCAGAAUU", RnaTranscription.ToRna("ACGTGGTCTTAA"));
+    }
+
+    [Fact(Skip = "Remove to run test")]
+    public void Correctly_handles_invalid_input_rna_instead_of_dna_()
+    {
+        Assert.Equal(null, RnaTranscription.ToRna("U"));
```

Comment:
Either way has it's pros and cons, but now that I think about other exercises they're mostly throwing exceptions, so for consistency it probably makes sense to head in that direction.